autonomi/src/client/high_level/files/archive_private.rs

```diff
@@ -114,7 +114,11 @@ impl PrivateArchive {
 
     /// Deserialize from bytes.
     pub fn from_bytes(data: Bytes) -> Result<PrivateArchive, rmp_serde::decode::Error> {
-        let root: PrivateArchiveVersioned = rmp_serde::from_slice(&data[..])?;
+        let root: PrivateArchiveVersioned = match serde_json::from_slice(&data[..]) {
```

Comment:
Does this mean old node can't support new format ?
this many cause a little bit chaos during upgrade ?

may mark this PR as BREAKING CHANGE ?

Reply:
Nodes do not read the content of the (encrypted) chunks this will be in.

Reply:
Ah, right, on node side, the concern is invalid, as node doesn't need to know about the detailed content of an archive-chunk.

However, the Client has to be updated to support new format.
Meanwhile, the old archives cann't be upgraded, and will be persisted in the network (somehow the same as to the standarize issue that Happybeing pointed out),
as any archive format upgrade will result in new chunk and ripple effect to folders (archive) holding it.

Hence, I think the PR shall at least have a Breaking Change label , and some more notes in the PR description, to highlight this?